configuration/configuration.go

```diff
@@ -363,6 +369,7 @@ func (c *Data) GetTemplateValues() (map[string]string, error) {
 		"CHE_KEYCLOAK_AUTH__SERVER__URL": c.GetKeycloakURL() + "/auth",
 		"CHE_KEYCLOAK_REALM":             c.GetKeycloakRealm(),
 		"CHE_KEYCLOAK_CLIENT__ID":        c.GetKeycloakClientID(),
+		"JENKINS_ROOT_URL":               c.GetOpenshiftJenkinsRootURL(),
```

Comment:
Need to add CM mapping in https://github.com/chmouel/fabric8-tenant/blob/master/OpenShiftTemplate.yml#L103 and some dummy data in the example: https://github.com/chmouel/fabric8-tenant/blob/master/f8tenant.secrets.yaml#L31